src/plato-mini/README.md

```diff
@@ -0,0 +1,29 @@
+## Jina X plato-mini
+
+## GetStart
+
+please install pdm first: `pip install pdm`
```

Comment:
Could you link the `pdm` to its home page, so that reader can learn more about it.